content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md

```diff
@@ -65,9 +65,13 @@ There are eight stages as shown below in this example pipeline.
 
    ![sonar-token](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/sonar-token.jpg)
 
-3. In total, you have four credentials in the list.
+3. You also need to create a GitHub personal access token with the permission as shown in the below image, and then use the generated token to create another Account Credentials, for example, `github-token`, in your DevOps project. 
 
-   ![credential-list](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/credential-list.jpg)
+   ![github-token-scope](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-a-jenkinsfile/github-token-scope.png)
+
```

Comment:
Add a note here:

To create a GitHub personal access token, go to **Settings** of your GitHub account, click **Developer settings**, select **Personal access tokens**, and click **Generate new token**.